Hello World example build fails

Hi, this looks really useful but the example build seems to fail out of the box.

```
error[E0308]: mismatched types
  --> examples/hello_world.rs:66:9
   |
66 |         &window,
   |         ^^^^^^^ expected struct `winit::window::Window`, found a different struct `winit::window::Window`
   |
   = note: expected reference `&winit::window::Window` (struct `winit::window::Window`)
              found reference `&winit::window::Window` (struct `winit::window::Window`)
   = note: perhaps two different versions of crate `winit` are being used?
```

It seems like quite a strange error as the types clearly match. I'm quite new to rust so the error may be on my part but any help would be appreciated. Thanks

Macos
rustc 1.49.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
 and this project adheres to cargo's version of [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - [Unreleased](#unreleased)
+- [v0.14.0](#v0140)
 - [v0.13.1](#v0131)
 - [v0.13.0](#v0130)
 - [v0.12.0](#v0120)
@@ -13,7 +14,16 @@ and this project adheres to cargo's version of [Semantic Versioning](https://sem
 
 ## Unreleased
 
+## v0.14.0
+
+Released 2021-02-12
+
+#### Updated
+- updated `imgui` to 0.7
+
 ## v0.13.1
+
+Released 2021-02-01
 
 #### Fixed
 - Readme
@@ -48,7 +58,8 @@ Released 2020-11-21
 
 ## Diffs
 
-- [Unreleased](https://github.com/Yatekii/imgui-wgpu-rs/compare/v0.13.1...HEAD)
+- [Unreleased](https://github.com/Yatekii/imgui-wgpu-rs/compare/v0.14.0...HEAD)
+- [v0.14.0](https://github.com/Yatekii/imgui-wgpu-rs/compare/v0.13.1...v0.14.0)
 - [v0.13.1](https://github.com/Yatekii/imgui-wgpu-rs/compare/v0.13.0...v0.13.1)
 - [v0.13.0](https://github.com/Yatekii/imgui-wgpu-rs/compare/v0.12.0...v0.13.0)
 - [v0.12.0](https://github.com/Yatekii/imgui-wgpu-rs/compare/v0.11.0...v0.12.0)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "imgui-wgpu"
-version = "0.13.1"
+version = "0.14.0"
 authors = ["Noah Hüsser <yatekii@yatekii.ch>", "Connor Fitzgerald <connorwadefitzgerald@gmail.com>", "Steven Wittens <steven@acko.net>"]
 edition = "2018"
 description = "A wgpu render backend for imgui-rs."
@@ -21,19 +21,18 @@ exclude = [
 
 [features]
 # used in src/simple_api.rs
-simple_api_unstable = [ "winit", "pollster", "imgui-winit-support" ]
-
 default = []
+simple_api_unstable = [ "winit", "pollster", "imgui-winit-support" ]
 
 [dependencies]
 bytemuck = "1"
-imgui = "0.6"
+imgui = "0.7"
 log = "0.4"
 smallvec = "1"
 wgpu = "0.7"
 
 # deps for simple_api_unstable
-imgui-winit-support = { version = "0.6", optional = true }
+imgui-winit-support = { version = "0.7", optional = true }
 pollster = { version = "0.2.0", optional = true } # for block_on executor
 winit = { version = ">= 0.23, < 0.25", optional = true }
 
@@ -42,13 +41,10 @@ bytemuck = { version = "1.4", features = ["derive"] }
 cgmath = "0.17"
 futures = "0.3"
 image = "0.23"
-imgui-winit-support = "0.6"
+imgui-winit-support = "0.7"
 raw-window-handle = "0.3"
 wgpu-subscriber = "0.1"
 winit = "0.24"
 
 [package.metadata.docs.rs]
 all-features = true
-
-# [patch.crates-io]
-# wgpu = { version = "0.4", git = "https://github.com/gfx-rs/wgpu-rs.git", rev = "7b15e24" }

--- a/README.md
+++ b/README.md
@@ -17,6 +17,6 @@ cargo run --release --example hello_world
 
 # Status
 
-Basic features are useable. Uses `wgpu-0.7.0` and `imgui-0.6.0` upstream. `winit-0.24` is used with the examples.
+Basic features are useable. Uses `wgpu-0.7.0` and `imgui-0.7.0` upstream. `winit-0.24` is used with the examples.
 
 Contributions are very welcome.

